R/rmItem.R

```diff
@@ -23,9 +23,11 @@
 rmItem <- function(dgeObj, itemName){
 
     assertthat::assert_that(class(dgeObj)[[1]] == "DGEobj",
-                            !missing(itemName),
+                            msg = "The DGEobj must be of class 'DGEobj'.")
+    assertthat::assert_that(!missing(itemName),
                             length(itemName) == 1,
-                            class(itemName)[[1]] == "character")
+                            class(itemName)[[1]] == "character",
+                            msg = "Be sure to specify a singular itemName as a character string.")
```

Comment:
Let's remove "Be sure" - Messages should be fairly austere, more functional than friendly :-)